CleanCode.SolidPrinciples/OpenClosed/Ocean.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CleanCode.SolidPrinciples.OpenClosed
+{
+    public class Ocean :ObservationPlace
+    {
+        public Ocean()
+        {
+            this.Type = PlaceType.Ocean;
+            this.IgnoredSightings = new[] { "~" };
+            this.Message = "bul bul bul";
+        }
+
+        public override bool ObservePlace()
```

Comment:
Nie jest to najlepsze podejście moim zdaniem. Mamy powielanie bardzo podobnego kodu jak Console.WriteLine(this.Message) itp. Mogłoby to się znajdować w ObservationPlace